### DISCUSSION TITLE
Select multiple formats for Transformations Presets

### DISCUSSION THREAD

**Discussion by flayks** [directus/directus#13698] [Feature Requests]
Do you think that it would be possible to avoid these situations of Transformation Presets where we need JPG and WebP for a size? 

Probably a way to select multiple Formats and not just one, as it really can bloat the presets and makes it hard to read/find. We could even think of a quick search input to find the one we need in case we have too many. I currently have 23 where I could only have 12 😮

I already tried to deselect the Format option but then Directus returns a JPG by default.

![image](https://user-images.githubusercontent.com/273716/171856974-be57491f-95fc-48ad-b8d5-2bdec55605bb.png)
![image](https://user-images.githubusercontent.com/273716/171856987-3d4e223d-2286-4bf7-b432-642171bcf747.png)

**Comment by rijkvanzanten** [comment]
I think the same would apply to any of the other configurations! Would be nice if you could "hardcode" some of those options, and leave others flexible (eg require size 500x200, but allow any format and quality, or require jpegs, but don't care about the size) 🤔

**Comment by flayks** [comment]
@rijkvanzanten are you still considering this option for allowing multiple options, more especially for the file formats?

**Reply by rijkvanzanten** [reply]
Absolutely! That being said, I do have to prioritize what I'm working on in my (shockingly) limited time, and this (unfortunately) hasn't gotten a lot of traction / love from the community, so it's difficult for me to prioritize this particular improvement :C 

Wanna help out with a PR? I think we might be able to configure those options with a "allow anything" type of wildcard (maybe like `"format": "*"`) which then signals to the API that it should use whatever the parameter is that was used still 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
